pages/common/gsutil.md

```diff
@@ -0,0 +1,33 @@
+# gsutil
+
+> The gsutil CLI is a Python application that lets you access Cloud Storage from the command line.
+> You can use gsutil to do a wide range of bucket and object management tasks.
+> More information: <https://cloud.google.com/storage/docs/gsutil>.
+
+- List all buckets in a project you are logged into:
+
+`gsutil ls`
+
+- List the objects in a bucket:
+
+`gsutil ls -r gs://{{BUCKET_NAME}}/{{PREFIX}}**`
```

Comment:
```suggestion
`gsutil ls -r gs://{{bucket_name}}/{{prefix}}**`
```

Reply:
We use lower case for placeholders. But there are some exceptions: placeholders with SQL code for instance.